code/sum.py

```diff
@@ -102,10 +103,8 @@ def download_video(url):
     returns:
         True
     '''
-    yt = YouTube(url)
-    yt.set_filename('1')
-    video = yt.filter('mp4')[-1]
-    video.download(os.getcwd())
+    #This download the video and download the subtitles and convert to srt
+    os.system('youtube-dl -f mp4 --write-auto-sub --convert-subtitles srt ' + url)
```

Comment:
What will be the filename for the video and subtitle file?
Ideally, it should be consistent such as `1.mp4` and `1.srt`